src/test/java/com/circlek/Test01.java

```diff
@@ -10,33 +10,41 @@
 
 public class Test01 {
 
-    //CODE REVIEW
+    //CODE REVIEW_Ilya
     @Test
-    public void assertTextOnPage(){
+    public void assertTextOnPage() {
         System.setProperty("webdriver.chrome.driver", "lib/chromedriver.exe");
         WebDriver driver = new ChromeDriver();
         driver.get("https://circlek-public.github.io/cki-service-recruitment-task/");
 
         //assert Remember Me Button
+
         WebElement rememeberMeLabel = driver.findElement(By.id("remember_me_label"));
+
         assertEquals("Remember Me", rememeberMeLabel.getText());
 
         //assert Sign In text
+
         WebElement signInButton = driver.findElement(By.className("login"));
+
         assertEquals("Sign In", signInButton.getText());
     }
 
     @Test
-    public void checkRememberMeCheckBox(){
+    public void checkRememberMeCheckBox() {
         System.setProperty("webdriver.chrome.driver", "lib/chromedriver.exe");
         WebDriver driver = new ChromeDriver();
         driver.get("https://circlek-public.github.io/cki-service-recruitment-task/");
 
         //click Remember Me checkbox
+
         WebElement rememberMeCheckBox = driver.findElement(By.xpath("//*[contains(text(), \"Remember Me\")]/../input"));
```

Comment:
1. As I mentioned above, it's bettor to replace it in a separate method in the PageObjectClass
2. xpath locator looks very unstable and difficult here, we can replace it 
```suggestion
        WebElement rememberMeCheckBox = driver.findElement(By.xpath("//input[@id = 'remember_me']"));
```
if we want to carry on using xpath or we can use it via more simple css or by id locators